scripts/metadata2html.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+import os
+import sys
+
+# CCPP framework imports
+from parse_tools import init_log, set_log_level
+from metadata_table import MetadataHeader
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--metafile', '-m', action='store',
+                    help='name of metadata file to convert',
+                    required=True)
+parser.add_argument('--outputdir', '-o', action='store',
+                    help='directory where to write the html files',
+                    required=True)
+
+attributes = [ 'standard_name', 'long_name', 'units', 'local_name',
```

Comment:
Which attributes should be displayed. I would argue not all of them in the scientific documentation. Maybe just standard_name, long_name, and units (intent might be useful too)?

Reply:
Correct. Something we need to iterate on with Ligia, Julie and Man as well. This script is a first version and I was hoping to improve it with the coming metadata conversions.